packages/applications-service-api/src/repositories/document.ni.repository.js

```diff
@@ -60,7 +61,11 @@ const buildWhereStatements = (caseReference, searchTerm, filters, datePublished)
 	];
 
 	if (searchTerm) {
-		const searchTermStatement = mapSearchTermToQuery(searchTerm);
+		const searchTermStatement = mapNISearchTermToQuery(searchTerm, [
+			'description',
+			'personal_name',
+			'representative'
```

Comment:
Need to add 'mine' here to I believe as it was here previously.

Reply:
ok, I'll put it back